distribution/docs/src/main/resources/_contents/_metrics/metrics-collection-contents.adoc

```diff
@@ -0,0 +1,16 @@
+
+==== Metrics Collection Application
+
+The Metrics Collection Application is responsible for collection both Catalog and Source metrics.
+
+Use Metrics Collection to collect historical metrics data, such as catalog query metrics, message latency, or individual sources' metrics type of data.
+
+===== Installing Metrics Collection
+
+The Metrics Collection application is installed by default with a standard installation.
+
+The catalog level metrics are packaged as the `catalog-core-metricsplugin` feature, and the source-level metrics are packaged as the `catalog-core-sourcemetricsplugin` feature.
```

Comment:
Revise: Need a dash to join `catalog` and `level` ... should be `catalog-level`. 
Note we're doing that for `source-level` in this line as well.